.golangci.yml

```diff
@@ -1,22 +1,21 @@
+version: "2"
 run:
-  timeout: 10m
-
-  # Run linters over integration tests
   build-tags:
     - integration
-
 linters:
-  disable-all: true # Disable defaults, then enable the ones we want
+  default: none
   enable:
+    - bodyclose
     - errcheck
-    - gosimple
+    - gosec
     - govet
     - ineffassign
     - staticcheck
-    - typecheck
     - unused
-    - bodyclose
-    - stylecheck
-    - gosec
-    - goimports
-    - gofmt
```

Comment:
Should we have a `formatters` section for enabling `goimports` and `gofmt`?
https://golangci-lint.run/product/migration-guide/#lintersenableformatter_name

Reply:
[Added formatters](https://github.com/Open-CMSIS-Pack/cbuild/pull/396/commits/dfc93160cee053d61e86669acfeabc15ea12597f)